packages/lib-classifier/src/components/Classifier/components/Banners/components/SubjectSetProgressBanner/SubjectSetProgressBanner.js

```diff
@@ -8,13 +8,15 @@ import Banner from '../Banner'
 counterpart.registerTranslations('en', en)
 
 function SubjectSetProgressBanner({ subject, workflow }) {
+  const setName = workflow?.subjectSet.display_name
```

Comment:
There's a chance `subjectSet` could be undefined on the model, so I think for safety, there should be default:
```suggestion
  const setName = workflow?.subjectSet?.display_name || ''
```

Reply:
Good point, but subject selection would fail if there was no subject set selected, so I don't think this banner renders until after the subject set is set on the workflow.